docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+services:
+
+  starknet-devnet:
+    image: shardlabs/starknet-devnet
+    healthcheck:
+      test: ["CMD", "nc", "-z", "localhost", "5050"]
```

Comment:
there is also a route `curl http://0.0.0.0:5050/is_alive` supposed to return `Alive!!!` when the devnet is ready

Reply:
I didn't know that, very nice